src/main/resources/site/parts/pictureCardLinks/pictureCardLinks.ts

```diff
@@ -46,14 +45,11 @@ function parsePictureCardLinks(
   pictureCardLinks = Array.isArray(pictureCardLinks) ? pictureCardLinks : [pictureCardLinks]
   return pictureCardLinks.reduce((acc, pictureCardLink, i) => {
     if (pictureCardLink) {
-      const href: string = getLinkTargetUrl(pictureCardLink.urlContentSelector) ?? pictureCardLink.href ?? ''
```

Comment:
We still need `href`, otherwise the part will have an empty path. I was referring to `pictureCardLink.href` references.